hs_discover/deploy.sh

```diff
@@ -0,0 +1,15 @@
+npm run build
```

Comment:
this is a convenience script for developers, who are actively working on Discover frontend code and need to join it to the Django for end-to-end testing. My layout on my laptop is as in this image, included. Your laptop, VirtualBox or other dev server may vary so your convenience script will need to adapt: 
![image](https://user-images.githubusercontent.com/45397838/94194556-da00ab00-fe7f-11ea-8ac3-0adef0b331f4.png)